Unix/configeditor/configeditor.cpp

```diff
@@ -336,16 +349,22 @@ int MI_MAIN_CALL main(int argc, const char** argv)
         size_t pos = line.find("=");
         if (line.empty() || pos == string::npos)
         {
-            cout << line << std::endl;
+        	if (opts.query.empty())
+        	{
+                cout << line << std::endl;
+        	}
             continue;
         }
-
+        
         // Parse the property and value from the line
         string prop = line.substr(0, pos);
         _TrimLeadingTrailingSpaces(prop);
         if (prop.empty())
         {
-            cout << line << std::endl;
+        	if (opts.query.empty())
```

Comment:
I definitely like the idea of a flag here. Also, please combine the flag with the other condition at line 362. It won't make a code optimization difference, but no reason not to do both conditions in a single `if` statement.